src/sentry/incidents/subscription_processor.py

```diff
@@ -358,6 +361,25 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
             metrics.incr("incidents.alert_rules.skipping_already_processed_update")
             return
 
+        if features.has(
+            "organizations:workflow-engine-m3-dual-write", self.subscription.project.organization
```

Comment:
this looks like it's using the flag we want to use for dual writing to the database. let's use a different flag for processing the results.